LCJava/src/main/java/tw/TrainRoute.java

```diff
@@ -0,0 +1,195 @@
+package tw;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.*;
+
+public class TrainRoute {
+
+    private class ResultMessage {
+        public static final String SUCCESS =  "SUCCESS";
+        public static final String FAILURE = "NO SUCH ROUTE";
+    }
+
+    private Map<String, Map<String, Integer>> graph;
+
+
+    public TrainRoute (String filename) {
+        this.graph = new HashMap<>();
+        this.initGraph(filename);
```

Comment:
initGraph can only be called once, right? can you just move the code in initGraph to constructor?